README.md

```diff
@@ -45,6 +47,15 @@ or you can install the latest development version from GitHub:
 remotes::install_github("stan-dev/cmdstanr")
 ```
 
+#### Installing CmdStan
+
+If you don't already have CmdStan installed then, in addition to installing the
+R package, it is also necessary to install CmdStan using CmdStanR's 
+`install_cmdstan()` function. A suitable C++ toolchain is also required. 
+Instructions are provided in the [_Getting started with
+CmdStanR_](https://mc-stan.org/cmdstanr/articles/cmdstanr.html) vignette. 
```

Comment:
Maybe mention that on Windows they can double check the toolchain is installed properly with check_cmdstan_toolchain()? I know this is a part of install_cmdstan so maybe redundant,